CRM/Logging/Schema.php

```diff
@@ -131,6 +131,15 @@ public function __construct() {
     while ($dao->fetch()) {
       $this->tables[] = $dao->TABLE_NAME;
     }
+    // Get any non standard table names used for custom groups.
+    // include these BEFORE the hook is called.
+    $customFieldDAO = CRM_Core_DAO::executeQuery("
+      SELECT DISTINCT table_name as TABLE_NAME FROM civicrm_custom_group
+      where table_name NOT LIKE 'civicrm_%';
```

Comment:
Interesting: The underscore is a wildcard character with LIKE, so this would skip some intended tables (e.g. one called `civicrmstuff`), except because line 129 includes those tables when it didn't intend to, it works out. So 🤷

Reply:
Right, I think you need to use a "regex" here ie:
```suggestion
      where table_name NOT LIKE 'civicrm\_%';
```
(see api/v3/System.utf8conversion)

Reply:
Yes just then a table like 'civicrm4ever' would get included twice but you could run array_unique on the results. Or could change line 129 to also do `\_`, but a problem would be that that would start excluding _non-custom-field_ tables that were being logged before if they had names like civicrm4ever.

Reply:
@demeritcowboy @mattwire that is the same WHERE as is used to include a few lines earlier so it's basically 

- step one include all tables where table_name LIKE 'civicrm\_%';
- step two include custom tables WHERE table_name NOT LIKE 'civicrm\_%';

I tried doing it as a union but the charset didn't match when ci ran

Reply:
(ie the WHERE clause only exists because we might as well not get ones we already have)

Reply:
Currently if you have table called civicrm4eva then it will be included regardless of whether it is a custom table or not - I don't want to change that and I don't think this will.

Reply:
Right that's why I gave the shrug - this PR as-is is the most consistent, with the downside the queries are technically wrong, but they complement each other.

Reply:
@demeritcowboy yeah & changing would create a functional change - which I don't want to do